target/linux/ramips/image/mt76x8.mk

```diff
@@ -1024,6 +1024,16 @@ define Device/xiaomi_mi-router-4a-100m-intl
 endef
 TARGET_DEVICES += xiaomi_mi-router-4a-100m-intl
 
+define Device/xiaomi_mi-router-4a-100m-intl-v2
+  IMAGE_SIZE := 12672k
+  DEVICE_VENDOR := Xiaomi
+  DEVICE_MODEL := Mi Router 4A
+  DEVICE_VARIANT := 100M International Edition V2
+  DEVICE_PACKAGES := kmod-mt7615e kmod-mt7663-firmware-ap
+  SUPPORTED_DEVICES += xiaomi,mir4a-100m-intl-v2
```

Comment:
Remove SUPPORTED_DEVICES as it's a new device.

Reply:
Thanks alot. Can you explain, why is it needed? I used 4a-100m-intl and 4a-100m devices. They have similar variable in their sections.

Reply:
This is used for compatibility with openwrt 19 or older version. Most devices supported by openwrt19 use different device strings than they do now. But it's a small nit for mir4a-100m because it used the new naming convention from the beginning. So, you can also remove it from other r4a-100m revisions.
https://github.com/openwrt/openwrt/blob/1da2e82c1182a3fd681da5760be96821213afadd/target/linux/ramips/image/mt76x8.mk#L469